firefox_puppeteer/tests/test_update_wizard.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_puppeteer.ui.update_wizard import UpdateWizardDialog
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestUpdateWizard(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        def opener(win):
+            self.marionette.execute_script("""
+              let updatePrompt = Components.classes["@mozilla.org/updates/update-prompt;1"]
+                                 .createInstance(Components.interfaces.nsIUpdatePrompt);
+              updatePrompt.checkForUpdates();
+            """)
+
+        self.dialog = self.browser.open_window(callback=opener,
+                                               expected_window_class=UpdateWizardDialog)
+        self.wizard = self.dialog.wizard
+
+    def tearDown(self):
+        try:
+            self.windows.close_all([self.browser])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_basic(self):
+        self.assertEqual(self.dialog.window_type, 'Update:Wizard')
+        self.assertNotEqual(self.browser.dtds, [])
+        self.assertNotEqual(self.browser.properties, [])
+
+    def test_elements(self):
```

Comment:
Ok, it will be updated with my next commit. I added two groups - one for button and another for the panels. Extra elements for panels are rare, so I will leave the lines as they are right now.